packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -83,20 +98,52 @@ export const useResizeHandle = (params: UseResizeHandleParams) => {
       handleRef.current?.setAttribute(key, String(value));
     });
 
-    onChange?.(currentValue.current);
+    // Make sure to only apply the value if it's not the initial value!
+    if (currentValue.current !== UNMEASURED) {
+      wrapperRef.current?.style.setProperty(
+        variableName,
+        `${currentValue.current}px`
+      );
+      onChange?.(currentValue.current);
+    }
   }, [getA11ValueText, maxValue, minValue, onChange, variableName]);
 
   // In case the maxValue or minValue is changed, we need to make sure we are not exceeding the new limits
   React.useEffect(() => {
-    setValue(currentValue.current);
-  }, [maxValue, minValue, updateElementsAttrs]);
+    if (currentValue.current === UNMEASURED) {
+      return;
+    }
+    currentValue.current = clamp(currentValue.current, minValue, maxValue);
+  }, [maxValue, minValue]);
```

Comment:
```suggestion
    if (currentValue.current === UNMEASURED || relative) {
      return;
    }
    currentValue.current = clamp(currentValue.current, minValue, maxValue);
  }, [maxValue, minValue, relative]);
```

From what I understand relative should completely ignore min/max values

Reply:
Another thought - how about changing the reference of the `clamp` in the entire function to do nothing when `relative` is set? that way you won't need to worry about conditional logic for clamps.... as I understand clamp should never happen with relative

Reply:
I ended up creating `clampWithMode` where the last parameter is `relative?: boolean`